README.md

```diff
@@ -1 +1,49 @@
-# Cyclone
\ No newline at end of file
+<h1 align="center">
+	<br>
+	<img width="400" src="docs/images/logo.jpeg" alt="cyclone">
+	<br>
+	<br>
+</h1>
+
+Cyclone is a cloud native workflow engine run in Kubernetes. It's implemented with Kubernetes CRD (Custom Resource Definition), with no extra database components required. Once deployed, you will get one cloud-scale workflow engine ready for your workflows.
+
+Cyclone defines each workflow stage as a pod, and assemble them to workflow by capturing dependencies between stages as DAG. 
+
+## Features
+- **Kubernetes Native**: Cyclone run on any Kubernetes clusters without any other dependencies.
+- **CI/CD**:  Cyclone supports CI/CD pipeline natively by templates.
```

Comment:
**Workflow Engine**: Supports CI/CD pipelines, AI pipelines and any other general workflows.